.github/workflows/full-help-docs.yml

```diff
@@ -18,8 +18,8 @@ jobs:
       # this looks goofy to get GITHUB_OUTPUT to work with multi-line return values;
       # see https://stackoverflow.com/a/74266196/249801
       run: |
-        cargo md-gen
-        raw_diff=$(git diff docs/soroban-cli-full-docs.md)
+        make generate-full-help-doc
+        raw_diff=$(git diff FULL_HELP_DOCS.md)
```

Comment:
Renamed the doc file to disconnect it from the term "soroban-cli" and refer to it with a consistent single term "full help docs" throughout the repo and elsewhere.